macros/miscs/marlin_compat.cfg

```diff
@@ -2,14 +2,17 @@
 gcode:
     LOAD_FILAMENT
 
+
 [gcode_macro M702]
 gcode:
     UNLOAD_FILAMENT
 
+
 [gcode_macro M600]
 gcode:
     CHANGE_FILAMENT
 
+
```

Comment:
Please keep this file unmodified for the moment. I prefer to do small PR with functionnalities and keep the typo and formating for a later PR